cvat/apps/engine/media_extractors.py

```diff
@@ -90,13 +90,26 @@ def _get_preview(obj):
         else:
             preview = obj
         preview.thumbnail(PREVIEW_SIZE)
+        orientation = preview._getexif().get(274, 1) if preview._getexif() else 1
+        if orientation in [3, 4]:
```

Comment:
The same comment about magic number, maybe to create a corresponding enumeration?

Reply:
@ActiveChooN ,  I agree that it is not a good idea to use a private method. Please use the public API.